mmtrack/models/track_heads/siamese_rpn_head.py

```diff
@@ -436,8 +439,7 @@ def get_bbox(self, cls_score, bbox_pred, prev_bbox, scale_factor):
 
         bbox_pred = bbox_pred.permute(1, 2, 3, 0).contiguous().view(4, -1)
         bbox_pred = bbox_pred.permute(1, 0)
-        anchors = bbox_cxcywh_to_xyxy(self.anchors)
-        bbox_pred = self.bbox_coder.decode(anchors, bbox_pred)
+        bbox_pred = self.bbox_coder.decode(self.anchors, bbox_pred)
```

Comment:
Be careful about the `permute` operation, since we changed the order of anchors. We need check the `permute` operation in `get_bbox` and `loss` functions in order to keep them consistent.